### PR TITLE
DBZ-4148 Reconfigure for connect extension

### PR DESCRIPTION
**Pull request by mdrillin**
Rework the docker-compose configuration for connect extension
- now using connect nightly image
- remove previously added artifacts which are no longer necessary

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -61,8 +61,8 @@ services:
     command: bash -c 'sleep 20 && mongo -u admin -p admin --authenticationDatabase admin dbzui-db-mongo:27017/inventory --eval "rs.initiate();"'
   dbzui-connect:
     container_name: dbzui-connect
-    build: 
-      context: connect
+    image: 
+      debezium/connect:nightly
```
Comment (rk3rn3r) [inline docker-compose.yml:65]: @mdrillin can you maybe also please add the env var to enable scripting? some SMTs are in that package and it is needed for some functionality of Debezium SMTs.
compare: https://github.com/debezium/debezium-ui/blob/main/backend/src/test/java/io/debezium/configserver/util/Infrastructure.java#L66

Comment (mdrillin) [reply docker-compose.yml:65]: @rk3rn3r - I amended the commit to add the scripting env var.  Here is a screenshot of the transformation types I'm getting now:
TransformTypes

Comment (rk3rn3r) [reply docker-compose.yml:65]: LGTM!

**Comment by rk3rn3r** [comment]
Merged. Thx @mdrillin!

**Pull request merged**